daemons/gptp/linux/src/daemon_cl.cpp

```diff
@@ -81,11 +82,12 @@ void print_usage( char *arg0 ) {
 		  "\t-G <group> group id for shared memory\n"
 		  "\t-R <priority 1> priority 1 value\n"
 		  "\t-D Phy Delay <gb_tx_delay,gb_rx_delay,mb_tx_delay,mb_rx_delay>\n"
-		  "\t-T force master (ignored when Automotive Profile set)\n"
```

Comment:
Sorry if this is a bit of a backtrack on one of my earlier comments, but it looks like gPTP supported setting the state when not using the automotive profile. Do this support still exist now that you've switched over the implementation to using your extPortConfig variable?

Reply:
Hi Eric,
Since we need to separate the externalPortConfiguration feature and the automotive profile, the origianl "-T force master" & "-L force slave" options are analogous to "set external port configuration to master/slave". So I add the option "EP for enabling the external port configuration" to replace the original "force master/slave". The "-V” option is now only responsible for enabling the automotive profile. Does this make sense?

Reply:
Yep, that sounds good to me